src/Coduo/PHPHumanizer/String/Humanize.php

```diff
@@ -4,6 +4,9 @@
 
 class Humanize
 {
+    const SEPARATOR_DASH = '-';
+    const SEPARATOR_UNDERSCORE = '_';
```

Comment:
I dont think we really need those constants. 
``public function __construct($text, $capitalize = true, $separator = '-')``
should be enough. In next release I would like to change this behavior anyway, I would like to not pass aything by default as a separator.